packages/components/src/toggle-group-control/types.ts

```diff
@@ -112,6 +112,10 @@ export type ToggleGroupControlProps = Pick<
 	 * The selected value.
 	 */
 	value?: ReactText;
+	/**
+	 * The selected value.
```

Comment:
```suggestion
	 * The initial value to be used when in uncontrolled mode.
```

or something like that?

Also this needs to be added in the README.

Reply:
Good point, done in https://github.com/WordPress/gutenberg/pull/50278/commits/9de1f76d34618093b764d98eed87235b33cac88a